apps/finance/app/src/components/NewTransfer/Deposit.js

```diff
@@ -64,7 +64,7 @@ const initialState = {
       loading: false,
     },
     error: NO_ERROR,
-    index: 0,
+    index: -1,
     value: '',
   },
 }
```

Comment:
@dizzypaty Is this the intended look of this warning message?

<img width="446" alt="Screen Shot 2019-07-29 at 1 10 40 PM" src="https://user-images.githubusercontent.com/4166642/62044176-8d312e00-b202-11e9-863d-c77fa091c9a0.png">

Reply:
Obviously not 😂

We've defined the warning component appearance here: https://www.figma.com/file/WN8Yuc544XoH5NosMBPclTBx/Info-and-warning-components?node-id=0%3A1

Reply:
Also, we don't display icons anymore and in case of displaying a title, it should use the text style `LABEL 2`. (Which is not the case on the designs for this screen).

<img width="1534" alt="FINANCE_TRANSFER" src="https://user-images.githubusercontent.com/7205369/62295656-d6d18100-b46d-11e9-86af-f0ba81ad7a4f.png">

Reply:
Just as a note, the `Info` update merged a few hours ago takes care of that, including the text style if `title` is defined:

```jsx
<Info mode="warning">
  This organization doesn’t have any tokens available to withdraw
</Info>
```

```jsx
<Info mode="warning" title="Action impossible">
  This organization doesn’t have any tokens available to withdraw
</Info>
```